src/components-examples/cdk/tree/cdk-tree-complex/cdk-tree-complex-example.html

```diff
@@ -0,0 +1,35 @@
+<mat-spinner *ngIf="areRootsLoading | async; else treeTemplate"></mat-spinner>
```

Comment:
If you have the bandwidth, it'd be great if the examples used Angular's latest control flow syntax https://angular.io/guide/control_flow

Reply:
done